requirements.txt

```diff
@@ -6,5 +6,5 @@ lalsuite
 ligo-common
 pycbc
 pandas
-sxs[ecosystem]
+sxs
```

Comment:
@prayush this PR is ready for merge. However, I wanted to check with you if changing `sxs[ecosystem]` to `sxs` in the requirements is okay. I made this change to reduce the total number dependencies (although I guess `pycbc` and `lalsuite` are contributing most of them). The tests don't really fail due to this.

Reply:
Yes, that is actually good to reduce. `sxs[ecosystem]` adds things like jupyter, matplotlib, etc, which are strictly not needed for this package to function.